src/option.c

```diff
@@ -308,11 +316,14 @@ static void option_parse_token(char ** argv, int * flag_parsed_next, int * requi
                 if(strlen(arg) > 1){
                   printf("Error, ignoring remainder of string for option %c (%s).\n", o->shortVar, o->longVar);
                 }
-                break;
-              }
+	      }
```

Comment:
just checking why you removed the break here?

Reply:
hmm that look accidental, will fix..